### DISCUSSION TITLE
type 0x04

### DISCUSSION THREAD

**Discussion by zvikaf** [atac/pychapter10#40] [General]
Excellent work,
trying to implement streaming view...
do to intend to add type 0x04 ( Computer-Generated F4 - Streaming Config (TMATS)) to the package ?

**Comment by mcferrill** [comment]
No immediate plans, but would welcome a PR to that effect!

**Reply by zvikaf** [reply]
sound good plan...  will continue and try to implement the ability to handle real time c10 data from multicast telemetry :-)
